compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -455,6 +456,22 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleStridedSlice *nod
   return std::make_unique<kernels::StridedSlice>(input, begin, end, strides, output, params);
 }
 
+std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleSqueeze *node)
+{
+  assert(node->arity() == 1);
+
+  const Tensor *input = getInputTensor(node->input());
+  Tensor *output = getOutputTensor(node);
+
+  SqueezeParams params{};
+  params.squeeze_dims_count = node->squeeze_dims().size();
```

Comment:
It should be checked that the number of dimensions does not exceed the size of `SqueezeParams::squeeze_dims`.

Reply:
okay, I will add a assert before it that checks the size of node->squeeze_dims() exceed the size of `SqueezeParam::squeeze_dims`(4)